### DISCUSSION TITLE
Working example of a clickable Pie Chart

### DISCUSSION THREAD

**Discussion by jaydeepn** [primefaces/community#1194] [PrimeFaces] [answered]
Hello,

Where can I find a working example of a clickable pie chart (PrimeFaces).

I did like to basically have a drill down view when a pie is clicked.

Jay

**Comment by melloware** [comment, marked as answer]
You need to check how to do it in Chart.js: https://stackoverflow.com/questions/26257268/click-events-on-pie-charts-in-chart-js

And then you can use PrimeFaces Extender to add that custom JS: https://primefaces.github.io/primefaces/13_0_0/#/components/charts?id=extender

**Reply by jaydeepn** [reply]
Where can I find working sample repositories that can be leveraged?

**Reply by melloware** [reply]
There aren't any you will have to piece this together yourself from the links I have provided.
